pkgs/development/python-modules/pytest-forked/default.nix

```diff
@@ -14,19 +14,14 @@ buildPythonPackage rec {
     sha256 = "6aa9ac7e00ad1a539c41bec6d21011332de671e938c7637378ec9710204e37ca";
   };
 
-  buildInputs = [ pytest setuptools_scm ];
```

Comment:
Should `pytest` remain in `buildInputs` since it's listed in `install_requires` but we support multiple versions of `pytest`?

Reply:
pytest should go in `propagatedBuildInputs` and `setuptools_scm` should go into nativeBuildInputs

Reply:
But that would break packages that use `pytest_xdist` and e.g. `pytest_4`.

Reply:
pytest_xdist>=2 only works with pytest>=6 anyway